stages/code-traceability/src/main/java/edu/kit/kastel/mcse/ardoco/core/codetraceability/informants/ArCoTLInformant.java

```diff
@@ -35,7 +35,7 @@ public void run() {
             }
         }
 
-        Node root = TraceLinkGenerator.getRoot();
+        Node root = TraceLinkGenerator.getRoot(); //TODO maybe add preprocessing
```

Comment:
What kind of preprocessing :)

Reply:
there is an overload of the method `TraceLinkGenerator.getRoot()` that accepts a `NameComparisonUtils.PreprocessingMethod‎`